streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -23,7 +23,13 @@
 import org.apache.kafka.streams.processor.StreamPartitioner;
 
 /**
- * KTable is an abstraction of a <i>changelog stream</i> from a primary-keyed table.
+ * {@link KTable} is an abstraction of a <i>changelog stream</i> from a primary-keyed table.
```

Comment:
Add one line after "primary-keyed table.": Each record in this stream is an update on the primary-keyed table with the record key as the primary key.

Since we use the word "update" in other docstrings in this class we need to make it clear what an "update" is.